gitbootcamp.rst

```diff
@@ -203,6 +203,33 @@ from the `core-workflow  <https://github.com/python/core-workflow>`_
 repository to backport the commit.
 
 
+.. _git_from_mercurial:
+
+Applying a Patch from Mercurial to Git
+--------------------------------------
+
+Scenario:
+
+- A Mercurial patch exists but there is no pull request for it.
+
+Solution:
+
+1. Download the patch locally.
+
+2. Create a new branch and switch to it.
+
+3. For Mac and Unix::
+
+   $ patch -p1 < /path/to/issueNNNN.patch
```

Comment:
I vaguely remember the discussion.  It could be that the other person wasn't as familiar with `git apply`?  I agree with @matrixise that it makes sense to recommend the git supplied command since (1) it should work for all supported platforms and (2) it has the potential to work better than plain `patch`; read `git help apply` for all the details.